nvflare/utils/cli_utils.py

```diff
@@ -28,6 +28,14 @@ def get_home_dir() -> Path:
     return Path.home()
 
 
+def get_package_root() -> Path:
+    """
+    Get the nvflare package root folder, e.g.
+        /usr/local/python/3.10/lib/python3.10/site-packages/nvflare
+    """
+    return pathlib.Path(__file__).parent.parent.absolute().resolve()
+
+
```

Comment:
there should be place that use this method?
should we include all those changes in another PR that does cherry-pick?

@chesterxgchen

Reply:
This is used to find the .so in xgb_client_runner.py.